odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -50,27 +196,44 @@ def fbp_op(ray_trafo, padding=True):
         `CircularConeFlatGeometry` : Approximate reconstruction, correct in
         limit of fan angle = 0 and cone angle = 0.
 
-        `HelicalConeFlatGeometry` : Very approximate.
+        `HelicalConeFlatGeometry` : Very approximate unless a
+        `tam_danielson_window` is used. Accurate with the window.
 
         Other geometries: Not supported
 
     padding : bool, optional
         If the data space should be zero padded. Without padding, the data may
         be corrupted due to the circular convolution used. Using padding makes
         the algorithm slower.
+    filter_type : string, optional
+        The type of filter to be used. The options are, approximate order from
+        most noise senstive to least noise sensitive: 'Ram-Lak', 'Shepp-Logan',
+        'Cosine', 'Hamming' and 'Hann'.
+    filter_cutoff : float, optional
+        Relative cutoff frequency for the filter, a scalar in the range (0, 1].
+        The normalized frequencies are rescaled so that they fit into the range
+        [0, filter_cutoff].
```

Comment:
Can you use this, or a simplified version of it, in the private helpers? Helps developers.